contents/preface.md

```diff
@@ -55,19 +55,19 @@ In these chapter you will learn how to:
 ## Software engineering and data science {#sec:engineering}
 
 Unlike most books on data science, this book lays more emphasis on **properly structuring code**.
-The reason for this is that we noticed that many data scientists simply place their code in one large file and run all the statements sequentially.
+The reason for this is that we noticed that many data scientists simply place their code into one large file and run all the statements sequentially.
 You can think of this as writing a book for someone and forcing that person to read it from beginning to start.
 This works great for small and simple projects, but the bigger or more complex the project becomes, more problems will start to arise.
 For example, with a well written book: the book is split into distinctly named chapters and sections which contains several references to other parts in the book.
 The software equivalent of this is **splitting code into functions**.
 Each function has a name and some contents.
 With functions, you can basically tell the computer at any point in your code to jump to some other place and continue from there.
-This allows you to more easily re-use code between projects, update code, share code, collaborate and see big picture.
+This allows you to more easily re-use code between projects, update code, share code, collaborate and see the big picture.
 Hence, with functions, you can **save time**.
 
-So, when reading this book, you eventually will get accustomed to reading and using functions.
```

Comment:
Thank you for all the well spotted mistakes.

For this comma, I don't think that it should be removed. It's a nonrestrictive clause, that is, "when reading this book" is not essential for the sentence.

Reply:
so, I'm not removing the comma per se, I'm suggestion a change(like saying that this will be a process):
 
`So, while reading this book you will eventually get accustomed to reading and using functions.`

Reply:
Lets not nitpick about this further :+1: